script/main.py

```diff
@@ -243,17 +244,37 @@ def average_age(conn, gender):
     print('Average age of women: {value:.2f} years'.format(value = average_age_of_women))
   else:
     print('Overall average age: {value:.2f} years'.format(value = average_age_overall))
+
+
+def select_all_sorted_cities(conn):
+  cur = conn.cursor()
+  cur.execute("SELECT location_city, count(location_city) FROM users GROUP BY location_city ORDER BY count(location_city) DESC")
+  cities = cur.fetchall() 
+  return cities
+
+
+def most_popular_cities(conn, number):
+  cities = select_all_sorted_cities(conn)
```

Comment:
Put the contents of this functions here. It is not worth to create another function just for SQL command